app/models/models_raffle.py

```diff
@@ -0,0 +1,64 @@
+"""Models file for module_tombola"""
+from sqlalchemy import Column, Enum, Float, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
+from app.database import Base
+from app.models.models_core import CoreGroup, CoreUser
+from app.utils.types.raffle_types import RaffleStatusType
+
+
+class Raffle(Base):
+    __tablename__ = "raffle"
+    id: str = Column(String, primary_key=True, index=True, nullable=False)
+    name: str = Column(String, nullable=False)
+    status: RaffleStatusType = Column(
+        Enum(RaffleStatusType), nullable=False, default=RaffleStatusType.creation
+    )
+    group_id: str = Column(ForeignKey("core_group.id"), index=True, nullable=False)
+    description: str = Column(String, index=True, nullable=True)
+
+    group: CoreGroup = relationship("CoreGroup")
+
+
+class TypeTicket(Base):
+    __tablename__ = "raffle_type_ticket"
+    id: str = Column(String, primary_key=True, index=True, nullable=False)
+    price: float = Column(Float, nullable=False)
+    pack_size: int = Column(Integer, nullable=False)
+    raffle_id: str = Column(ForeignKey("raffle.id"), index=True, nullable=False)
+
+    raffle: Raffle = relationship("Raffle")
+
+
+class Lots(Base):
```

Comment:
I'm not sure lot is a valid English word, maybe Prize would be better.

Reply:
Will be fixed later (for compatibility with frontend)